### PR TITLE
add getters to BaseInstallObject

### PR DESCRIPTION
**Pull request by andreicek**
Some styling fixes here and there. Wanted to help 🐑

### CODE REVIEW

File: lib/generate-install-object/install-object/base.js [hunk 1]
```diff
@@ -39,12 +39,12 @@ class BaseInstallObject {
    */
   toString() {
     if (this.source === sources.REGISTRY) {
-      return `${this.name}${this.getVersionSuffix()}`;
+      return `${this.name}${this.versionSuffix}}`;
```
Comment (DarkoKukovec) [inline lib/generate-install-object/install-object/base.js]: You have an extra `}`

File: lib/get-dependency-examples/utils.js [hunk 2]
```diff
@@ -10,8 +10,9 @@ import { EXAMPLE_DIRECTORIES, MODULES } from '../consts';
  * @returns {Array} A list of dependency example paths
  */
 export const getExampleDirectoryPaths = function (dependencySetup) {
-  return EXAMPLE_DIRECTORIES
-    .map((exampleDirectory) => path.join(MODULES, dependencySetup.toLocalName(), exampleDirectory));
+  return EXAMPLE_DIRECTORIES.map(
```
Comment (DarkoKukovec) [inline lib/get-dependency-examples/utils.js]: Why format map like this? If you need to chain something to it, it wouldn't be readable

Comment (andreicek) [reply lib/get-dependency-examples/utils.js]: Line got too long. What do you think about:

```
return EXAMPLE_DIRECTORIES
  .map(
    (exampleDirectory) => path.join(MODULES, dependencySetup.toLocalName(), exampleDirectory)
  );

**Review by DarkoKukovec — APPROVED** [review]

**Comment by andreicek** [comment]
🎉

**Pull request merged**